src/index.ts

```diff
@@ -2,12 +2,15 @@ import { ApolloServer } from '@apollo/server';
 import express from 'express';
 import http from 'http';
 import cors from 'cors';
+import cookieParser from 'cookie-parser';
```

Comment:
Looks like you're missing the "types" defs for cors and cookie-parser. Just need to run
 ```
npm i --save-dev @types/cors
 @types/cookie-parser
```